base/CMakeLists.txt

```diff
@@ -348,7 +348,7 @@ add_subdirectory(util)
 add_subdirectory(common)
 add_subdirectory(tools)
 
-if(WITH_SERVER OR WITH_CA OR WITH_KRA OR WITH_OCSP OR WITH_TKS OR WITH_TPS OR WITH_ACME)
+if(WITH_SERVER OR WITH_CA OR WITH_KRA OR WITH_OCSP OR WITH_TKS OR WITH_TPS OR WITH_ACME OR WITH_EST)
```

Comment:
Are there dependency with the server code? The package in the liste do not compile is the server is not built. Is the same for EST?